raiden-dapp/src/components/transaction-history/Transaction.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="transaction">
+    <v-row class="transaction__item" no-gutters>
+      <v-col class="transaction__item__icon" cols="1">
+        <v-img
+          v-if="transfer.direction === 'sent'"
+          height="45px"
+          width="45px"
+          :src="require('../../assets/sent_transfer.svg')"
+        />
+        <v-img
+          v-else
+          height="45px"
+          width="45px"
+          :src="require('../../assets/received_transfer.svg')"
+        />
+      </v-col>
+      <v-col class="transaction__item__details-left">
+        <v-row no-gutters>
+          {{
+            transfer.direction === 'sent'
+              ? $t('transfer-history.sent-title')
+              : $t('transfer-history.received-title')
+          }}
+          <address-display
+            class="transaction__item__details-left__address"
+            :address="transfer.partner"
+          />
+        </v-row>
+        <v-row class="transaction__item__details-left__time-stamp" no-gutters>
+          {{ new Intl.DateTimeFormat('en-US').format(transfer.changedAt) }}
```

Comment:
this feels like something that would be a nice filter https://vuejs.org/v2/guide/filters.html